src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  background: $white;
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+  max-width: 200px;
+  &__image {
+    width: 100%;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
```

Comment:
Verify that the image `imac.jpeg` exists in the `../images/` directory and that the path is correct in your SCSS file.